src/elm/Update.elm

```diff
@@ -107,7 +109,7 @@ updateResolver : RM.Msg -> Model -> ( Model, Cmd Msg )
 updateResolver msg model =
     let
         token =
-            Maybe.withDefault "" <| currentToken model
+            Maybe.withDefault (Token.fromString "") <| currentToken model
```

Comment:
Is an empty token even a thing?

Reply:
It's not, but I wanted to avoid the larger refactor here (basically, we'd align the messages and routes in a way where we're guaranteed a token).